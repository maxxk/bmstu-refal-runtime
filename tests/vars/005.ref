$ENTRY Go
{
	= <Prout <Func (asd)>> <Prout <Func (asd) rt qw yuio 34 2 'sdqwe'>>
}

Func
{
	v.1 = v.1
}



