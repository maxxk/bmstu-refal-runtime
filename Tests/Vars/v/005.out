(asd )
(asd )rt qw yuio 34 2 sdqwe
